src/contracts/utils/address-utils.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.5.1;
```

Comment:
Give credit to https://github.com/OpenZeppelin/openzeppelin-solidity/blob/f7e53d90fa638553ffc93e93fe1b12fc081bb774/contracts/utils/Address.sol